src/reynir/reynir.py

```diff
@@ -496,6 +533,36 @@ def __init__(self, **options):
             Greynir constructor """
         self._options = options
 
+    @staticmethod
+    def _dump_token(tok):
+        """ This is here so that ReynirCorrect can override how tokens are dumped """
+
+        # Should look like this in ReynirCorrect
+        # def _dump_token(tok):
+        #     if hasattr(tok, '_err'):
+        #         err = tok._err
+        #         if isinstance(err, bool):
+        #             return tuple(*tok, err)
+        #         else:
+        #             return tuple(*tok, {
+        #                 **err.__dict__,
+        #                 'class_name': err.__class__.__name__,
+        #             })
+
+        #     return tuple(tok)
+
+        return tuple(tok)
+
+    @staticmethod
+    def _load_token(kind, txt, val):
+        """ Load token from serialized data """
+        val = [(BIN_Meaning(*v) if isinstance(v, list) else v) for v in val]
```

Comment:
The test here should probably just be for `kind == TOK.WORD`, that's the only token type having `BIN_Meaning` lists in the `val` field. However, `TOK.PERSON` also has a list (of `PersonName` tuples) in its `val` field - does that work out OK?

Reply:
Yeah, I think it does.